backend/views/syson-diagram-common-view/src/main/java/org/eclipse/syson/diagram/common/view/services/ViewCreateService.java

```diff
@@ -950,4 +956,31 @@ private boolean isPart(Element element) {
     private boolean isAction(Element element) {
         return element instanceof ActionUsage || element instanceof ActionDefinition;
     }
+
+    /**
+     * Creates a new sibling part usage as well as a subsetting edge between those parts.
+     *
+     * @param self
+     *         the {@link PartUsage} to subset by a new part usage
+     * @return the new part usage or self if something went wrong.
+     */
+    public Object createPartUsageAndSubsetting(Element self) {
+        if (self instanceof PartUsage partUsage) {
```

Comment:
It seems this service can only be applied on `PartUsage`. So could you check that the following signature: `public PartUsage createPartUsageAndSubsetting(PartUsage self)` is ok? Among your tests, please verify that the backend console does not display an AQL error with this new signature.

Reply:
Done.
There are no messages when the signature is changed.